docs/install.md

```diff
@@ -77,7 +77,7 @@ You may deploy the EBS CSI driver via Kustomize, Helm, or as an [Amazon EKS mana
 
 #### Kustomize
 ```sh
-kubectl apply -k "github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-1.37"
+kubectl apply -k "github.com/kubernetes-sigs/aws-ebs-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-1.38"
```

Comment:
*1.38.1

Reply:
Release branches are per minor version, not patch

Reply:
I don't think the patch version is needed for kustomize deployment. Hence why for 1.37.0 all that was needed is 1.37.

Reply:
Gotcha thank you resolving comment